agent/result_parser.py

```diff
@@ -42,13 +42,15 @@ def parse_results(results: whois.parser.WhoisCom) -> Iterator[Dict[str, Any]]:
                 "expiration_date": get_isoformat(
                     scan_output_dict.get("expiration_date", [])
                 ),
-                "name": name,
+                "name": _format_str(name) if name is not None else name,
```

Comment:
The fix should be in `_format_str` to just return None if the value is None, otherwise, the code will fail, if anywhere else we have the same issue.